packages/fuel-indexer-tests/indexers/fuel-indexer-test/fuel_indexer_test.yaml

```diff
@@ -4,7 +4,9 @@ graphql_schema: packages/fuel-indexer-tests/indexers/fuel-indexer-test/schema/fu
 abi: packages/fuel-indexer-tests/contracts/fuel-indexer-test/out/debug/fuel-indexer-test-abi.json
 start_block: ~
 end_block: ~
-contract_id: fuel12zfku575m48qgctuca7lf5rxwnrv93nn3yfxxtdte9hqqj7v45vsuujmpt
+contract_id:
+  - fuel1p4g0suykga85824l4x465haxtg7n6rlefxv2tch8av5g6ry88pxszp6aj0
+  - 0x0d50f87096474f43aabfa9abaa5fa65a3d3d0ff94998a5e2e7eb288d0c87384d
```

Comment:
Let's remove `fuel1...` and just use `0x0d5...` - and if tests pass, that proves this works :)